### DISCUSSION TITLE
Change the Docker scripts to pull latest versions

### DISCUSSION THREAD

**Discussion by cmar7945** [tteck/Proxmox#3934] [Ideas]
Is there a reason the scripts involving docker dont pull the latest version of docker.io or docker compose/portainer? that causes some issues when they arent set up for latest automatically. I usually have to manually go in and change the image tags to keep them updated. would be nice for that to be done automatically by the scrip on install :/

**Comment by tteck** [comment]
Which is not starting out using the latest?
https://github.com/tteck/Proxmox/blob/9a0cb6b8f5a379a0f4ecbcad8c2e77fc36709e2f/install/docker-install.sh#L35
https://github.com/tteck/Proxmox/blob/9a0cb6b8f5a379a0f4ecbcad8c2e77fc36709e2f/install/docker-install.sh#L49
https://github.com/tteck/Proxmox/blob/9a0cb6b8f5a379a0f4ecbcad8c2e77fc36709e2f/install/docker-install.sh#L70

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/blob/9a0cb6b8f5a379a0f4ecbcad8c2e77fc36709e2f/install/docker-install.sh#L22-L29

**Comment by cmar7945** [comment]
assuming this applies to the Docker LXC setup as well? which is marekd for the docker.sh script

**Reply by tteck** [reply]
As you can see, those snippets are from the Docker LXC (`docker-install.sh`).

**Reply by cmar7945** [reply]
This is the installer I look at for the LXC. is that the same? 
`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/docker.sh)"`

**Reply by tteck** [reply]
Nope, https://github.com/tteck/Proxmox/blob/main/CODE-AUDIT.md.
